rakelib/user_email_identifier.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'roo'
+
+desc 'Build a CSV of user email identifiers'
+
+namespace :user_email_identifier do
+  task :build_csv, %i[uuid_list] => :environment do |_, args|
+    xlsx = Roo::Excelx.new(args[:uuid_list])
+    xlsx.sheet(0)
+    row_count = xlsx.last_row - 1
+    puts "[UserEmailIdentifier] Building CSV from #{args[:uuid_list]}, processing #{row_count} rows..."
+
+    CSV.open('./tmp/user_email_identifiers.csv', 'w') do |csv|
+      csv << %w[data_type form_type va_gov_submission_created_at va_gov_user_uuid va_gov_credential_email]
+      xlsx.each_row_streaming(offset: 1, pad_cells: true) do |row|
+        next if row[3].nil? || row[3].cell_value.nil?
+
+        original_columns = row[0..3].map { |cell| cell && cell.cell_value.to_s.strip }
+        csp_uuid = original_columns[3]
+        type = csp_uuid.scan(/-/).empty? ? 'idme' : 'logingov'
+        verification = UserVerification.find_by_type!(type, csp_uuid)
```

Comment:
You'll want to 'find' by the `backing_credential_identifier`, since I believe this uuid could be based off an 'idme backed mhv' or 'idme backed dslogon' account